### PR TITLE
testutils: expose underlying channel in Channel

### PR DESCRIPTION
**Pull request by jackwener**
### What problem does this PR solve?

Issue Number: #4662

### What is changed and how it works?

What's Changed:

Expose underlying channel in Channel.

### Related changes

None

Side effects

- None

RELEASE NOTES: N/A

### CODE REVIEW

**Comment by jackwener** [comment]
@easwars PTAL

File: internal/testutils/channel.go [hunk 1]
```diff
@@ -25,20 +25,22 @@ import (
 const DefaultChanBufferSize = 1
 
 // Channel wraps a generic channel and provides a timed receive operation.
+// C is the underlying channel on which values sent using the SendXxx() methods are delivered.
+// Tests which cannot use ReceiveXxx() for whatever reasons can use C to read the values.
```
Comment (easwars) [inline internal/testutils/channel.go:29]: Please move the comment to be above the `C` field.

**Review by easwars — CHANGES_REQUESTED** [review]
Were you also planning to fix the test mentioned in the original issue description?

**Comment by jackwener** [comment]
> Were you also planning to fix the test mentioned in the original issue description?

Yes, it's ok.

**Comment by easwars** [comment]
> > Were you also planning to fix the test mentioned in the original issue description?
> 
> Yes, it's ok.

Sorry, did you mean to say you were going to fix the test as well? If you planned to do in a separate PR, then it would be better to just do it here.

**Pull request closed without merging**